src/OgreVideoManager.cpp

```diff
@@ -105,16 +111,27 @@ namespace Ogre
 		TheoraVideoClip* clip=createVideoClip(new OgreTheoraDataStream(mInputFileName,group_name),TH_RGBA,0,1);
 		int w=nextPow2(clip->getWidth()),h=nextPow2(clip->getHeight());
 
-		TexturePtr t = TextureManager::getSingleton().createManual(name,group_name,TEX_TYPE_2D,w,h,1,0,PF_BYTE_RGBA,TU_DYNAMIC_WRITE_ONLY);
+		TexturePtr t = TextureManager::getSingleton().createManual(name,group_name,TEX_TYPE_2D,w,h,1,0,PF_X8R8G8B8,TU_DYNAMIC_WRITE_ONLY);
 		
 		if (t->getFormat() != PF_X8R8G8B8) logMessage("ERROR: Pixel format is not X8R8G8B8 which is what was requested!");
 		// clear it to black
 
+#if OGRE_VERSION_MAJOR >= 2 && OGRE_VERSION_MINOR >= 1
+		unsigned char* texData=(unsigned char*) t->getBuffer()->lock(v1::HardwareBuffer::HBL_DISCARD);
```

Comment:
rather than duplicating the whole line, add a `using v1::HardwareBuffer;` at the top of the file.